### PR TITLE
completed

### PR DESCRIPTION
**Pull request by Hussein1710**

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,44 +1,41 @@
...
-import * as AiIcons from "react-icons/ai"
-import { Link } from 'react-router-dom'
-import SidebarData  from "../SidebarData"
-import "./Navbar.css"
-
+import React, { useState } from "react";
+import * as FaIcons from "react-icons/fa";
+import * as AiIcons from "react-icons/ai";
+import { Link } from "react-router-dom";
+import { SidebarData } from "./SidebarData";
```
Comment (EmekaNkwo) [inline src/components/Navbar.jsx:5]: Where is the sidebarData file? Did you push it?

File: src/App.jsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import { BrowserRouter as Router } from 'react-router-dom'
 import Navbar from './components/Navbar'
 
 function App() {
   return (
     <Router>
        <Navbar />
-       <Switch>
+       {/* <Switch>
```
Comment (EmekaNkwo) [inline src/App.jsx:10]: I thought you have configured the routing.
I noticed there are no pages or you are still working on them?

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -1,44 +1,41 @@
-import React, { useState } from 'react'
-import * as FaIcons from "react-icons/fa"
-import * as AiIcons from "react-icons/ai"
-import { Link } from 'react-router-dom'
-import SidebarData  from "../SidebarData"
-import "./Navbar.css"
-
+import React, { useState } from "react";
+import * as FaIcons from "react-icons/fa";
```
Comment (EmekaNkwo) [inline src/components/Navbar.jsx:2]: Why are you importing all the icons inside "react-icons/fa", when it's only one icon you need, won't this make your project heavy?

File: src/components/Navbar.jsx [hunk 4]
```diff
@@ -1,44 +1,41 @@
...
-            <Link to="#" className='menu-bars'>
-              <FaIcons.FaBars onClick={showSidebar} />
+    <div className="sidebar">
+      <div className="navbar">
+        <Link to="#" className="menu-bars">
+          <FaIcons.FaBars onClick={showSidebar} />
+        </Link>
+      </div>
+      <nav className={sidebar ? "nav-menu active" : "nav-menu"}>
+        <ul className="nav-menu-items">
```
Comment (EmekaNkwo) [inline src/components/Navbar.jsx:19]: was toggling of the sidebar included in the Figma design?

**Review by Hussein1710 — COMMENTED** [review]
I'm done.

**Pull request merged**